src/components/Avatar.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="group relative w-24 h-24">
+    <img v-if="imgPath" :src="imgPath" alt="" class="w-24 h-24 rounded-full" />
+    <div
+      v-if="!imgPath"
+      class="
+        w-24
+        h-24
+        relative
+        flex
+        justify-center
+        items-center
+        rounded-full
+        bg-gray-500
+        text-6xl text-white
+        uppercase
+      "
+    >
+      {{ giveName() }}
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    firstName: String,
+    imgPath: String,
+  },
+  setup(props) {
+    function giveName() {
```

Comment:
raczej arrow function przypisane do consta i wtedy return

Reply:
a najlepiej computed -> https://medium.com/notonlycss/the-difference-between-computed-and-methods-in-vue-js-9cb05c59ed98